.gitignore

```diff
@@ -222,4 +222,7 @@ dmypy.json
 # USER DEFINED
 *.pdf
 
-junit/
\ No newline at end of file
+junit/
+Ubuntu_GitHub
+Ubuntu_GitHub.pub
```

Comment:
What are these lines here for?

If you do really need them we can leave them in there.

Reply:
Local SSH keys that git wanted to push.

Reply:
Oh, wow, weird place for them to sit. Yes definitely ignore those. ... but also maybe don't put the in the repo at all?

Reply:
Did better with this, next time around.

Reply:
I still would like to know why private keys are being placed inside the repo directory at all...

Reply:
I think I that was my error in creating them. The command line prompted for a file name. The website suggested not entering one. I read the website less closely than that command line and did enter a destination, which then entered the folder.